packages/terra-clinical-onset-picker/docs/UPGRADEGUIDE.md

```diff
@@ -1,5 +1,25 @@
 # Terra Clinical Onset Picker Upgrade Guide
 
+## Changes from 3.0 to 4.0
+
+### Breaking Change
+* Condensed onChange function props into a single prop (`onsetOnChange`) that returns an Object for the onsetDate.
+
+## New Feature
+
+* Added support for age units.
```

Comment:
What are the added support? How does someone leverage this?

Reply:
Added more details 77f1bc1